apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.Snippets/AspNetCoreSnippets.cs

```diff
@@ -49,6 +49,15 @@ public void Configure(IApplicationBuilder app)
         {
             // Use before handling any requests to ensure all unhandled exceptions are reported.
             app.UseGoogleExceptionLogging();
+
+            app.UseStaticFiles();
```

Comment:
Is the order of these calls important? If so, let's call that out in comments.

Reply:
Kinda but not really unless you had some overlap in routes in static files and mvc.  This is just how they order them in templates.